src/ModalDialog.js

```diff
@@ -40,7 +40,6 @@ export default class ModalDialog extends React.Component {
   static propTypes = {
     onClose: PropTypes.func, // required for the close button
     className: PropTypes.string, // css class in addition to .ReactModalDialog
-    width: PropTypes.number, // width
```

Comment:
@perrin4869 Can you change this to something like `width: PropTypes.oneOfType([PropTypes.number, PropTypes.string])`